crates/rpc/src/v02/method/call.rs

```diff
@@ -2,11 +2,7 @@ use crate::felt::RpcFelt;
 use crate::{context::RpcContext, v03::method::common::base_block_and_pending_for_call};
 use pathfinder_common::{BlockId, CallParam, CallResultValue, ContractAddress, EntryPoint};
 
-crate::error::generate_rpc_error_subset!(
-    CallError: BlockNotFound,
-    ContractNotFound,
-    ContractError
-);
+crate::error::generate_rpc_error_subset!(CallError: BlockNotFound, ContractNotFound, ContractError);
```

Comment:
This seems to be unrelated -- was it a non-current-stable version of `cargo fmt` that produced this change? Mine does not seem to reformat this line.